src/index.js

```diff
@@ -0,0 +1,124 @@
+function setUp(){
+    const $update = document.querySelector('#update');
+    $update.onclick = updateRates;
+
+    getFxRates();
+    setDate();
+}
+
+function updateRates(){
+    removeRates();
+    const [base, date] = readBaseAndDate();
+    removeBaseOptions();
+    getFxRates(base, date);
+}
+
+function getFxRates(base = 'EUR', date = 'latest'){
+    const URL = `https://api.exchangeratesapi.io/${date}?base=${base}`;
+    fetch(URL)
+        .then( response => response.json() )
+        .then( response => {showRates(response);} )
```

Comment:
No hacen falta las `{}`
```suggestion
        .then( response => showRates(response) )
```